doc/cookbook/source/examples/regression/kernel_ridge_regression.rst

```diff
@@ -0,0 +1,55 @@
+=======================
+Kernel Ridge Regression
+=======================
+
+Kernel ridge regression is a kernel-based regularized form of regression which learns a function in the space induced by the respective kernel and the data by minimizing a squared error loss with :math:`L_2` regularization. For linear ridge regression, it boils down to solving a linear system:
```

Comment:
Kernel ridge regression is a non-parametric form of ridge regression. The aim is to learn a function in the space induced by the respective kernel $k$ by minimizing a squared loss with a squared norm regularization term